packages/compass-e2e-tests/helpers/commands/connect-with-connection-string.ts

```diff
@@ -71,7 +71,7 @@ export async function connectWithConnectionString(
 
     // some connection should be expanded (ie. connected) by now
     await browser
-      .$(`${Selectors.SidebarTreeItems} [aria-expanded=true]`)
-      .waitForExist({ reverse: true });
```

Comment:
how was this working until now?

Reply:
If your selector matches nothing then waitForExist({ reverse: true }) cannot fail :)